include/motion_control/pregenerated_motion_profile.h

```diff
@@ -0,0 +1,39 @@
+#ifndef MOTION_CONTROL_PREGENERATED_MOTION_H
+#define MOTION_CONTROL_PREGENERATED_MOTION_H
+
+#include <motion_control/motion_profile_1d.h>
+#include <cstdlib>
+
+namespace MotionControl {
+
+template<const int steps>
+class PregeneratedMotionProfile {
+    ProfilePoint1D points[steps];
```

Comment:
Move the private member to the bottom.

```
class TypeName {
public:
    ...
private:
    ...
}
```

(See pitt-ras/motion-doc@9b28a1d23edc9bc07e86ccc91932832cf635f0f4)